website/docs/working-with-containers/registries/_insecure-registry-windows-macos.md

```diff
@@ -0,0 +1,44 @@
+#### Prerequisites
+
+- [A running Podman machine](/docs/onboarding-for-containers/installing-podman).
+
+#### Procedure
+
+1. Add your insecure registry within **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
+
+   ![Adding a custom registry](img/adding-a-custom-registry.png)
+
+1. Click "Yes" to the insecure registry warning.
+
+   ![Podman Desktop Registry Warning](img/registry-warning-insecure.png)
```

Comment:
false positive on vale issue?

Reply:
Bad rule. I opened an issue. And am currently lost in regex: do not match strings starting by img/. The `/docs...` notation breaks the build...